storage/pkg/flagset/storagepubliclink.go

```diff
@@ -13,53 +13,39 @@ func StoragePublicLink(cfg *config.Config) []cli.Flag {
 			Name:        "debug-addr",
 			Value:       "0.0.0.0:9179",
 			Usage:       "Address to bind debug server",
-			EnvVars:     []string{"STORAGE_STORAGE_PUBLIC_LINK_DEBUG_ADDR"},
+			EnvVars:     []string{"STORAGE_PUBLIC_LINK_DEBUG_ADDR"},
 			Destination: &cfg.Reva.StoragePublicLink.DebugAddr,
 		},
 
 		&cli.StringFlag{
 			Name:        "network",
 			Value:       "tcp",
 			Usage:       "Network to use for the storage service, can be 'tcp', 'udp' or 'unix'",
-			EnvVars:     []string{"STORAGE_STORAGE_PUBLIC_LINK_NETWORK"},
-			Destination: &cfg.Reva.StoragePublicLink.Network,
-		},
-		&cli.StringFlag{
-			Name:        "protocol",
-			Value:       "grpc",
-			Usage:       "protocol for storage service, can be 'http' or 'grpc'",
-			EnvVars:     []string{"STORAGE_STORAGE_PUBLIC_LINK_PROTOCOL"},
-			Destination: &cfg.Reva.StoragePublicLink.Protocol,
+			EnvVars:     []string{"STORAGE_PUBLIC_LINK_GRPC_NETWORK"},
+			Destination: &cfg.Reva.StoragePublicLink.GRPCNetwork,
 		},
 		&cli.StringFlag{
 			Name:        "addr",
 			Value:       "0.0.0.0:9178",
 			Usage:       "Address to bind storage service",
-			EnvVars:     []string{"STORAGE_STORAGE_PUBLIC_LINK_ADDR"},
-			Destination: &cfg.Reva.StoragePublicLink.Addr,
-		},
-		&cli.StringFlag{
-			Name:        "url",
-			Value:       "localhost:9178",
-			Usage:       "Address to bind storage service",
-			EnvVars:     []string{"STORAGE_STORAGE_PUBLIC_LINK_URL"},
-			Destination: &cfg.Reva.StoragePublicLink.URL,
+			EnvVars:     []string{"STORAGE_PUBLIC_LINK_GRPC_ADDR"},
+			Destination: &cfg.Reva.StoragePublicLink.GRPCAddr,
 		},
 
 		&cli.StringFlag{
 			Name:        "mount-path",
-			Value:       "/public/",
+			Value:       "/public",
```

Comment:
double check if public links work. I also updated the registry to use a path without slash: https://github.com/owncloud/ocis/pull/657/files?file-filters%5B%5D=.go#diff-76cf344e66e9654cf08805e489b5d6c2R230